### ISSUE TITLE
Test for WCAG success criterion 1.3.3: Sensory Characteristics

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#182]
http://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-understanding.html
Instructions provided for understanding and operating content do not rely solely on sensory characteristics of components such as shape, size, visual location, orientation, or sound.
## Selector

texts
## Failures
- [ ] F14: Identifying content only by its shape or location ( #185 )
- [x] F26: Using a graphical symbol alone to convey information (#183 )
## Techniques
- [ ] G96: Providing textual identification of items that otherwise rely only on sensory information to be understood 
## Relationship between techniques and failures
- F14 is the counterpart of G96
- F26 is also covered by H37 if it's an image
